app/lua53/lnodemcu.c

```diff
@@ -579,6 +577,10 @@ LUAI_FUNC int  luaN_reload_reboot (lua_State *L) {
 #endif
   lua_settop(L, 1);
   lua_getglobal(L, "file");
+  if (lua_isnil(L, 2)) {
+    lua_pushstring(L, "No file system mounted");
```

Comment:
Mounted?  Wouldn't this be more like "file module not compiled into firmware"?

Any particular reason this is returned rather than thrown?